### ISSUE TITLE
REQUESTS_CA_BUNDLE cannot be set to a directory

### ISSUE THREAD

**Issue by psliwka** [aws/aws-cli#3425]
```sh
$ aws --version
aws-cli/1.15.31 Python/2.7.14+ Linux/4.16.0-2-amd64 botocore/1.10.31
```
On my system, I am using some internal CA certificates, installed to `/etc/ssl/certs`. Therefore, I exported `REQUESTS_CA_BUNDLE=/etc/ssl/certs` to my global environment variables, so that all Python apps using `requests` can see my certs. With most apps, everything works just fine.

However, this does not work for `aws-cli`. Attempt to run any command results in following error:
```sh
$ aws --endpoint=https://<my.service>:<port> s3api head-bucket --bucket <my-bucket>

[Errno 21] Is a directory
```

To make things work, I need to specify path to particular CA cert file, such as `REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt`.

**Comment by stealthycoin** [comment]
First off I would suggest not relying on `REQUESTS_CA_BUNDLE` since the fact that the CLI is using requests is an implementation detail and could change, though we specifically at the moment honor that environment variable Its not documented. I would suggest using `AWS_CA_BUNDLE` instead since that is something we support explicitly and won't change.

The CLI doesn't really do anything special in this case. It just forwards this as the verify argument to our old vendored version of requests which sends it the vendored version of urllib3 which does not support the directory format when it verifies the cert. It looks to me like it was added here: https://github.com/urllib3/urllib3/commit/75e071eea2d0698b0089db13a2b716a8f5f18eb1 which according to the tags means its supported in `>=1.12` and we have vendored `1.10` so this isn't available. I will mark it as a feature request.

But if you use `AWS_CA_BUNDLE` and point right at a .crt the CLI will use that, and ignore `REQUESTS_CA_BUNDLE` which can be a directory for the rest of your apps.
